spec/features/create_weather_spec.rb

```diff
@@ -1,30 +1,40 @@
 require 'rails_helper'
+require 'vcr'
 
-feature 'User adds a new weather', %Q{
-  As an authenticated user
-  I want to add a weather
-  So that others can review it
-  } do
+  feature 'User adds a new weather', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Inconsistent indentation detected.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.